CITATION.cff

```diff
@@ -1,71 +1,90 @@
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
-
+# YAML 1.2
+# Metadata for citation of this software according to the CFF format (https://citation-file-format.github.io/)
 cff-version: 1.2.0
-message: "If you use this software, please cite it as below."
+message: If you use this software, please cite it using these metadata.
+title: 'STEllAR-GROUP/hpx: HPX V1.7.0: The C++ Standards Library for Parallelism and
+  Concurrency'
+doi: 10.5281/zenodo.5185328
 authors:
-- family-names: "Kaiser"
-  given-names: "Hartmut"
-  orcid: "https://orcid.org/0000-0002-8712-2806"
-- family-names: "Diehl"
-  given-names: "Patrick"
-  orcid: "https://orcid.org/0000-0003-3922-8419"
-- family-names: "Lemoine"
-  given-names: "Adrian S."
-- family-names: "Adelstein Lelbach"
-  given-names: "Bryce"
-  orcid: "https://orcid.org/0000-0002-7995-5226"
-- family-names: "Amini"
-  given-names: "Parsa"
-  orcid: "https://orcid.org/0000-0002-6439-8404"
-- family-names: "Berge"
-  given-names: "Agust�n"
-- family-names: "Biddiscombe"
-  given-names: "John"
-  orcid: "https://orcid.org/0000-0002-6552-2833"
-- family-names: "Brandt"
-  given-names: "Steven R."
-  orcid: "https://orcid.org/0000-0002-7979-2906"
-- family-names: "Gupta"
-  given-names: "Nikunj"
-  orcid: "https://orcid.org/0000-0003-0525-3667"
-- family-names: "Heller"
-  given-names: "Thomas"
-  orcid: "https://orcid.org/0000-0003-2620-9438"
-- family-names: "Huck"
-  given-names: "Kevin"
-  orcid: "https://orcid.org/0000-0001-7064-8417"
-- family-names: "Khatami"
-  given-names: "Zahra"
-  orcid: "https://orcid.org/0000-0001-6654-6856"
-- family-names: "Kheirkhahan"
-  given-names: "Alireza"
-  orcid: "https://orcid.org/0000-0002-4624-4647"
-- family-names: "Reverdell"
-  given-names: "Auriane"
-  orcid: "https://orcid.org/0000-0002-5531-0458"
-- family-names: "Shirzad"
-  given-names: "Shahrzad"
-  orcid: "https://orcid.org/0000-0001-9496-8044"
-- family-names: "Simberg"
-  given-names: "Mikael"
-  orcid: "https://orcid.org/0000-0002-7238-8935"
-- family-names: "Wagle"
-  given-names: "Bibek"
-  orcid: "https://orcid.org/0000-0001-6619-7115"
-- family-names: "Wei"
-  given-names: "Weile"
-  orcid: "https://orcid.org/0000-0002-3065-4959"
-- family-names: "Zhang"
-  given-names: "Tianyi"
-  orcid: "https://orcid.org/0000-0002-1000-4887"
-title: "HPX - The C++ Standard Library for Parallelism and Concurrency"
-doi: 10.21105/joss.02352
-version: 1.7.0
-date-released: 2021-07-14
-repository-code: "https://github.com/STEllAR-GROUP/hpx"
-url: "https://hpx.stellar-group.org"
-license: "BSL V1.0"
+- given-names: Hartmut
+  family-names: Kaiser
+- given-names: Mikael
+  family-names: Simberg
+- given-names: Bryce
+  family-names: Adelstein Lelbach
+  affiliation: NVIDIA
+- given-names: Thomas
+  family-names: Heller
+- given-names: Agustin
+  family-names: Berge
+- given-names: John
+  family-names: Biddiscombe
+  affiliation: CSCS
+- given-names: Auriane
+  family-names: R.
```

Comment:
```suggestion
  family-names: Reverdell
```

Sorry for noticing this only now. Feel free to just apply this directly on github. Let's *actually* merge this after that.

Reply:
Thanks :)

Reply:
Sorry for that, the Python script does a horrible jobs with the names. We need to be careful with updating the CFF file with each release.